dotCMS/src/main/java/com/dotcms/rest/api/v1/experiments/ExperimentsResource.java

```diff
@@ -213,6 +238,10 @@ private Experiment patchExperiment(final Experiment experimentToUpdate,
             builder.scheduling(experimentForm.getScheduling());
         }
 
+        if(experimentForm.getGoals()!=null) {
+            builder.goals(experimentForm.getGoals());
+        }
+
         return builder.build();
     }
 
```

Comment:
do we need a end point to add a goal too?

Reply:
We are good with PATCHing the experiment and including the goal. I have a postman test for that.